docs/sources/whatsnew/whats-new-in-v11-1.md

```diff
@@ -0,0 +1,171 @@
+
+![An xy chart showing height weight distribution](/media/docs/grafana/panels-visualizations/screenshot-xy-charts-v11.0.png)
+
+Over the past several months we’ve introduced multiple enhancements to the visualizations like auto mode, which now handles most scenarios that previously required manual configuration. Additionally, we've added better control over point styling and further improved performance. We’re excited to include XY chart as a first class citizen in the core Grafana visualization library. To learn more about the panel, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations/xy-chart/).
+
+To use the new XY chart visualization, you must first enable the `autoMigrateXYChartPanel` [feature toggle](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/).
+
+## Alerting
+
+<!-- TO DO - Confirm with Alerting folks that everything available in OSS is also available in Enterprise except for Rule specific silences with permissions and Rule specific silences with RBAC-->
+
+### Re-designed settings page for Alerting
+
+<!-- Gilles De Mey -->
+
+_Generally available in all editions of Grafana_
+
+The new settings page provides you with a holistic view of where Grafana-managed alert instances are forwarded.
+
+- Manage which Alertmanagers receive alert instances from Grafana-managed rules without navigating and editing data sources.
+- Manage version snapshots for the built-in Alertmanager, which allows administrators to roll back unintentional changes or mistakes in the Alertmanager configuration.
+- There is also a visual diff that compares the historical snapshot with the latest configuration to see which changes were made.
+
+{{< figure src="/media/docs/alerting/alert-settings.png" alt="Alert settings page" >}}
+
+### Alerting template selector
+
+<!-- Sonia Aguilar -->
+
+_Generally available in all editions of Grafana_
+
+Enables you to easily choose which templates you want to use in your alert notification messages by adding a template selector in the Contact Points form.
+
+Select from existing templates or enter a custom one for your specific needs. You can switch between the two tabs to access the list of available templates and copy them across to the customized version.
+
+### Add OAuth2 to HTTP settings for vanilla Alertmanager / Mimir
+
+<!-- Gilles de Mey -->
+
+_Generally available in all editions of Grafana_
+
+Configure OAuth2 authentication for any Alertmanager or Mimir receiver (called Contact Points in Grafana) through the user interface.
+
+OAuth2 is not implemented for the Grafana built-in Alertmanager.
+
+### Improved paused alert visibility
+
+<!-- Tom Ratcliffe -->
+
+_Generally available in all editions of Grafana_
```

Comment:
@tomratcliffe - Can you please confirm that this is available in both OSS **and** Enterprise? It was only listed as being available in Cloud and OSS on the original Cloud entry.

Reply:
Available in both OSS and Enterprise